src/site/apt/examples/set-compiler-release.apt.vm

```diff
@@ -106,3 +111,34 @@ Setting the <<<--release>>> of the Java Compiler
   [...]
 </project>
 +-----
+
+* Usage on JDK 8 - automatic detection
+
+  Since version <<<3.13.0>>> of the Compiler Plugin and default <<<javac>>> compilerId you can skip special profile.
```

Comment:
```suggestion
  Since version <<<3.13.0>>> of the Compiler Plugin together with the default <<<javac>>> compilerId you don't need conditional parametrisation of `release`.
```